docs/aop/user/apps/usercentrics.md

```diff
@@ -0,0 +1,138 @@
+
+{% info_block infoBox "Info" %}
+
+All of the three approaches require you to get the setting ID for your store in the [Usercentrics Admin Interface](https://admin.usercentrics.eu/) and configure the data processing services on page **Service Settings -> Data Processing Services**.
+
+{% endinfo_block %}
+
+#### Direct integration
+
+The direct integration requires mostly manual developer work. You can do the direct integration if you don't use a third-party tracking tool yet.
+
+The direct integration presupposes that you inject the Usercentrics JavaScrip tag into your site with the setting ID.
+	
+Example script tag for the direct integration:
+	
+```
+<script id="usercentrics-cmp" data-settings-id="apXtrZ9ec" src="https://app.usercentrics.eu/browser-ui/latest/bundle.js" defer></script>
+```
+
+where `apXtrZ9ec` is the setting ID that identifies configuration of the data processing services for your store in the Usercentrics Admin Interface.
+
+For the direct integration, you have to programmatically deactivate the JavaScript code of all the tracking tools on the shop pages and give tools the names that match those in the Usercentrics Admin Interface.
+   
+For more details on the Usercetrics direct integration, see the [Usercentrics documentation](https://docs.usercentrics.com/#/direct-implementation-guide).
+
+#### Smart Data Protector
+
+The [Smart Data Protector](https://docs.usercentrics.com/#/smart-data-protector) blocks the data processing services you added to your store and activates them only for customers that gave their consent to do so. The Smart Data Protector is the preferred and default setting to configure Usercentrics. You can apply it if you don't use a third-party tracking tool yet and don't want to use developers' work.
+
+For this setting, Usercentrics JavaScript tag has to be injected with the setting ID and Smart Data Protector JavaScript code. Once you set up data processing services and got the setting ID for your app from the Usercentrics Admin Interface, the Smart Data Protector automatically manages all the tracking tools, and no manual work is required from your side.
+
+{% info_block infoBox "Info" %}
+
+Every data processing service not supported by Usercentrics by default requires a custom manual configuration in the Usercentrics Admin Interface and corresponding adaption as for the [Direct Integration](#direct-integration) case. For details on the custom services, see [Usercentrics documentation on custom data processing services](https://usercentrics.atlassian.net/servicedesk/customer/portal/2/article/185794627).
+
+{% endinfo_block %}
+
+#### Google Tag Manager
+
+If you already have the [Google Tag Manager](https://developers.google.com/tag-platform/tag-manager) integrated inещ your store, to use the technology legally, you need the cookie consent popup to appear on your Storefront. You can achieve that by using the Google Tag Manager with Usercentrics.
```

Comment:
Here we need a check by our developer Igor Galayev, to potentially add some info / context that the user centrics of the Google Tag Manager documentation was missing out on, so that there are no big gaps.

Reply:
In general it looks pretty good
I left a few comments